usr/share/rear/layout/save/GNU/Linux/220_lvm_layout.sh

```diff
@@ -123,7 +128,7 @@ local lvs_exit_code
         # so that when $vgrp is empty 'test $vgrp -a $pdev' tests if file $pdev exists
         # which is usually true because $pdev is usually a partition device node (e.g. /dev/sda1)
         # so that when $vgrp is empty 'test $vgrp -a $pdev' would falsely succeed:
-        test $vgrp && test $pdev || Error "LVM 'lvmdev' entry in $DISKLAYOUT_FILE where volume_group or device is empty or more than one word"
+        test $vgrp && test $pdev || LogPrint "WARNING: LVM 'lvmdev' entry in $DISKLAYOUT_FILE where volume_group or device is empty or more than one word"
```

Comment:
Better omit the word `WARNING` cf.
https://schlomo.schapiro.org/2015/04/warning-is-waste-of-my-time.html
in particular in this case it is always false alarm because
when $vgrp is empty the entry was disabled above
so I think the `test $vgrp` is useless here and should be removed
with comment and user message adapted accordingly.